dist/providers/google/redirect.js

```diff
@@ -3,7 +3,7 @@ import { ConfigError } from "../../utils/errors";
 export default async function redirect({ options, }) {
     const { clientId, redirectUrl, // Deprecated, use redirectTo instead
     redirectTo, // Use this instead of redirectUrl
-    scope = "openid email profile", responseType = "code", state = "pass-through value", accessType = "online", } = options;
+    scope = ["openid", "email", "profile"], responseType = "code", state = "pass-through value", accessType = "online", } = options;
```

Comment:
If we change this, we would need to change the scope type string to array as well. Also might create a different type then other providers?